### PR TITLE
Added ping ping to keep alive the ws connection. #16

### PR DESCRIPTION
**Pull request by pablisho**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -40,11 +43,27 @@ def get(self):
...
+    def __init__(self, application, request, **kwargs):
+        tornado.websocket.WebSocketHandler.__init__(self, application, request, **kwargs)
+        self.io_loop = tornado.ioloop.IOLoop.instance()
+
     def open(self):
         global clients_connected, authenticate, proxy, clients
         clients_connected += 1
         print "Client connected.  %d clients total." % clients_connected
         clients.append(self)
+        self.io_loop.add_timeout(datetime.timedelta(seconds=random.randint(5,30)), self.send_ping)
```
Comment (adamantivm) [inline main.py]: Can you please make the interval configurable?

**Comment by pablisho** [comment]
@adamantivm I made it configurable the same way it's the "port" right now, getting from the environment.., but I don't if that's ok and I don't know how to set that in heroku either.

**Comment by adamantivm** [comment]
@pablisho I think reading from an environment variable is an OK choice. [Here](https://devcenter.heroku.com/articles/config-vars) you can read how to set them on Heroku.

**Comment by adamantivm** [comment]
I see this also includes passing the URL parameters for the video feed to the client as arguments. I request please next time send different features in separate pull requests (note this PR doesn't have any reference on the title or description about this new feature being included)

**Pull request merged**